components/Layout/index.tsx

```diff
@@ -129,10 +129,7 @@ export const Layout: React.VFC<{
               logoAltText: tsln.logoAltText,
             }}
           />
-          <h1
-            id="applicationTitle"
-            className="font-header-gc font-[700] text-[#333333] text-[38px] leading-[42px] mb-8 mt-4 sm:mt-12 sm:w-[100%]"
-          >
+          <h1 id="applicationTitle" className="h1 my-8">
```

Comment:
Looks good to me, just check with Andreanne about the font size
this is from prod

![image](https://github.com/DTS-STN/eligibility-estimator/assets/52539578/79a0bfbc-feaa-4ccd-b498-94381d9d0b44)

Reply:
versus
![image](https://github.com/DTS-STN/eligibility-estimator/assets/52539578/1fa1d612-a08a-482f-a1ef-b279777a5ebd)

Reply:
> Andreanne

H1 font size in mobile is different, updated.